Rakefile

```diff
@@ -111,7 +111,7 @@ namespace :build do
       env_vars = ''
       env_vars += 'MOOC_LOCALIZE=1 ' if CDO.localize_apps
       env_vars += 'MOOC_DIGEST=1 ' unless rack_env?(:development)
-      RakeUtils.system "#{env_vars} grunt"
+      RakeUtils.system "#{env_vars} grunt build"
```

Comment:
I did investigate the scss issue and it no longer repros. Let's leave this as `grunt build` and if we run into any problems with scss changes not showing up in apps-package, we can revisit.